modules/home/src/lib/latest-news/latest-news.component.scss

```diff
@@ -0,0 +1,116 @@
+@import '../../../../../src/scss/global';
```

Comment:
Acredito que podemos remover esse import se utilizarmos a flag `stylePreprocessorOptions`. Depois dá uma conferida [nesse artigo](https://dev.to/danywalls/how-to-simplify-and-clean-your-sass-import-with-angular-json-57pg) 😉